config/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "esModuleInterop": true,
+    "resolveJsonModule": true
+  }
+}
```

Comment:
I found this article https://skovy.dev/code-splitting-with-webpack-typescript-react/ on having a separate `tsconfig.json` when writing Webpack configs in Typescript. Although it feels dirty, it makes sense. What you think about splitting the common chunks as well, as mentioned in "Optimizing Split Chunks" in that article?

Reply:
That's the article I used as a reference actually! Common chunks is something I could look at in the future, for sure.